frontend-react/src/hooks/UsePaginator.tsx

```diff
@@ -64,7 +64,7 @@ function usePaginator(
     /* Handles adding cursors in the Map when responseArray changes */
     useEffect(() => {
         const nextCursor =
-            responseArray[filterState.pageSize - 1]?.createdAt || null;
+            responseArray[filterState.pageSize - 1]?.timestamp || null;
```

Comment:
I changed 3 fieldnames in the submissions collection/list response to be consistent with how they are named in the History API.  Changed in both back and front ends.